### ISSUE TITLE
Status ?

### ISSUE THREAD

**Issue by Xample** [altab-ch/ionic-plugin-datatrans#1]
Hi, the title is interesting but what is the current status of this package ?
I am developing an ionic app and as datatrans works with twint, it seems to be a possible solution.

**Comment by altab-ch** [comment]
Hi @Xample, I started this project as I wanted to develop an application for small business.
But Covid arrived and my project had to be stoped.

**Comment by gsttg** [comment]
> Hi, the title is interesting but what is the current status of this package ? I am developing an ionic app and as datatrans works with twint, it seems to be a possible solution.

@Xample did you find a solution for datatrans + ionic? (I am looking for something like that at the moment)

**Comment by Xample** [comment]
No, I ended up using payrexx. But there was no native plugin at the time I did it. I therefore had to implement a web view for the payment.

**Comment by gsttg** [comment]
Thank you!
